Gems/LmbrCentral/Code/Source/Shape/EditorReferenceShapeComponent.h

```diff
@@ -8,24 +8,24 @@
 
 #pragma once
 
-#include <Components/ReferenceShapeComponent.h>
-#include <Vegetation/Editor/EditorVegetationComponentBase.h>
+#include<LmbrCentral/Component/EditorWrappedComponentBase.h>
```

Comment:
nit: missing space beween `include` and  `<`

Reply:
Fixed